templates/zerver/emails/email.css

```diff
@@ -84,65 +86,52 @@ table td {
     margin-top: 20px;
 }
 
-.footer td,
-.footer span {
+.content-block,
```

Comment:
The selector `.footer td` is equivalent to `.content-block`, while `.footer span` corresponds to `.apple-link`.